README.md

```diff
@@ -99,6 +99,12 @@ Options:
         --seed SEED                  Seed for rspec
 ```
 
+To pass any options supported by paralell_tests, use `--` :
+
+```bash
+bundle exec turbo_tests -n 4 -- --only-group 1 --pattern spec/system
+```
+
```

Comment:
Let's also add an example on how to use other formatters like fuubar (ref: https://github.com/serpapi/turbo_tests/pull/51#issuecomment-2049699960)

Reply:
Done !